physionet-django/events/templates/events/event_participation.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <h5>Event Registration</h5>
+  {% if registration_allowed %}
+    <strong><p>Click the button below to join the waiting list. You will receive a notification when your registration is confirmed.</p></strong>
```

Comment:
Change "Click the button below to join the waiting list. You will receive a notification when your registration is confirmed.: to:

```
Click the button below to register for the event. You will receive a notification when your registration is confirmed.
```